src/components/ex-download/ex-download.scss

```diff
@@ -0,0 +1,77 @@
+.download {
+  @apply mt-component-spacing-small w-full;
+
+  &__no-title {
+    @apply flex items-center px-4 py-3 font-bold text-white;
+    background-color: rgba(65, 131, 215, 1);
+
+    &:before {
+      @apply mr-2;
+      content: '\26A0';
+    }
+  }
+
+  &_item {
+    @apply flex flex-col w-full px-4 py-2;
+
+    background: #f5f8fa;
+  }
+
+  &_image {
+    @apply hidden;
+  }
+
+  &_title {
+    @apply font-body text-default text-gray-1 flex items-center justify-start flex-initial m-0 font-normal leading-5 underline;
+    &:hover {
+      @apply text-primary;
+    }
+    &::before {
+      @apply flex-none block mr-2 bg-center bg-no-repeat bg-contain;
+      content: '';
+      width: 18px;
+      height: 20px;
+      background-image: url('./resources/images/sicherearbeit-file-icon.svg');
+      @screen md {
+        @apply mr-4;
+      }
+    }
+  }
+  &_description {
+    @apply hidden;
+  }
+  &_button {
+    @apply hidden;
+  }
+  &.download-preview {
```

Comment:
The gray does not match the one of the other boxes. Same goes for font colors I believe and definitely the underlines.

![image](https://user-images.githubusercontent.com/88790097/134132698-31808d44-25ee-4e8e-b84e-c068931ffe22.png)

Reply:
changed the color